.vscode/extensions.json

```diff
@@ -0,0 +1,9 @@
+{
+  "recommendations": [
+    "ms-vscode-remote.vscode-remote-extensionpack",
+    "ms-vscode-remote.remote-containers",
+    "ms-azuretools.vscode-docker",
+    "GitHub.vscode-pull-request-github",
```

Comment:
This extension facilitates the inline commit diff_ing_; basically the `gh` CLI but as a GUI in VSCode.